### ISSUE TITLE
Screen reader does not provide the information about the sorting under table controls.

### ISSUE THREAD

**Issue by sereymonyros** [microsoft/fluentui#13232]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: (fill this out if relevant)

#### Actual behavior:

 https://developer.microsoft.com/en-us/fluentui/#/controls/web/detailslist

#### Expected behavior:

 Screen reader should be able to speak sort acceding/descending. 

### Priorities and help requested:

Are you willing to submit a PR to fix? (Yes, No)

Requested priority: (Blocking, High, Normal, Low)

Products/sites affected: (if applicable)

**Comment by ecraig12345** [comment]
You can already do this by providing `IColumn.sortedAscendingAriaLabel` and `IColumn.sortedDescendingAriaLabel`, as shown with the Name column on the page you linked. It will be spoken after the title of the column. 

If this isn't working for you, please provide the specific browser and OS version you're using, and the specific steps for reproing the problem. (In general, please fully fill out the template when filing an issue, rather than skipping some parts.)
